eslint.config.js

```diff
@@ -72,87 +58,19 @@ const config = [
                     fixStyle: "separate-type-imports",
                 },
             ],
-            "@typescript-eslint/consistent-type-exports": [
-                "error",
-                {
-                    fixMixedExportsWithInlineTypeSpecifier: false,
-                },
-            ],
+            "@typescript-eslint/consistent-type-exports": "error",
             "@typescript-eslint/no-import-type-side-effects": "error",
             "@typescript-eslint/no-unused-vars": [
                 "error",
                 {
                     ignoreRestSiblings: true,
                 },
             ],
-            "comma-dangle": ["error", "always-multiline"],
-            "comma-spacing": [
-                "error",
-                {
-                    before: false,
-                    after: true,
-                },
-            ],
-            "comma-style": ["error", "last"],
-            "eol-last": ["error", "always"],
```

Comment:
I believe all these removed entries are for rules that are superseded by Prettier - and that they happen to match your Prettier settings. But since Prettier is already formatting the files, there's no need to ask ESLint to as well. Here and later, except for `no-unused-vars` and the `react/*` ones.